traja/tests/test_nn.py

```diff
@@ -7,7 +7,7 @@
 
 data_url = "https://raw.githubusercontent.com/traja-team/traja-research/dataset_und_notebooks/dataset_analysis/jaguar5.csv"
 df = pd.read_csv(data_url, error_bad_lines=False)
-model_save_path = './model.pt'
+model_save_path = 'model'
```

Comment:
Or you recommend to create dir "model" to save the model as "model.pt" by default?

Reply:
That file is replaced by of test_model.py - issue fixed there!